src/cfn.py

```diff
@@ -189,12 +190,15 @@ def _user(use_bootstrap_user):
 
 @task
 @requires_aws_stack
-def cmd(stackname, command=None):
+def cmd(stackname, command=None, username=DEPLOY_USER):
     if command is None:
         abort("Please specify a command e.g. ./bldr cmd:%s,ls" % stackname)
-    with stack_conn(stackname):
-        with settings(abort_on_prompts=True):
-            run(command)
+    print "Connecting to: %s" % stackname
```

Comment:
In places where multiple instance support is needed, usages of `with stack_conn(stackname)` become calls to `stack_all_ec2_nodes` passing a function to repeatedly execute.